README.md

```diff
@@ -73,9 +84,9 @@ in your browser. Note that Beehive will create a config file `beehive.conf`
 in its current working directory, unless you specify a different file with the
 `-config` option.
 
-Note: You currently have to start `beehive` from within $GOPATH/src/github.com/muesli/beehive
-in order for it to find all the resources for the admin interface. Also see the
-Troubleshooting & Notes section of this README.
+**Note**: you currently have to install Beehive in `$GOPATH/src/github.com/muesli/beehive`
```

Comment:
That's not actually true: you can install Beehive from any directory, if you know what you're doing. The important part is running it from the folder with the checkout git submodules, so it can find the HTML/JS sources.

Reply:
@muesli the user can also rewrite Beehive from scratch, if he knows what he is doing. :-) If I had difficulty and I have been using GNU Git for years, I think it is appropriate to give simple instructions: if the end user "knows what he does", then he will guess that he can make changes.
As always in programming the worst case prevails: the user does not know how git works or how Beehive's dependencies work.

Reply:
@muesli then? It's been 2 days, do you want to merge or not?